src/scala/ly/stealth/mesos/kafka/Broker.scala

```diff
@@ -67,8 +67,11 @@ class Broker(_id: String = "0") {
 
     result.putAll(options)
 
-    if (bindAddress != null)
+    if (bindAddress != null) {
       result.put("host.name", bindAddress.resolve())
+      if (result.get("listeneres") != null)
```

Comment:
@steveniemitz 
Really sorry it's typo. I updated code again with unit tests added for "listeners" and "bindAddress".
Thanks for comments.